src/grlib/load_data/with_labels_loader.py

```diff
@@ -23,20 +23,21 @@ def create_landmarks(self, labels: pd.DataFrame, output_file='landmarks.csv'):
         Images are labelled according to provided *labels*
         If no hand is found, all 63 landmarks are set to 0
         :param labels: a dataframe with additional image path (no dataset root) in *path* column and *label* column.
-        If no *labels* column is provided, landmarks are computed, but no labels are assigned.
+        If no *label* column is provided, landmarks are computed, but no labels are assigned.
         :param output_file: the file path of the file to write to
         :return: None
         """
         files = self.path + labels['path'] + '.jpg'
```

Comment:
Maybe we should replace calls like these with `os.path.join`. iirc that method will automatically check if `self.path` already has a slash at the end, and if not it will add it.